### ISSUE TITLE
Bind the RETURN key to the WPM and Tone Entry Fields

### ISSUE THREAD

**Issue by callej** [callej/NATO-Morse-2#5]
### Allow RETURN to enter the value in the Entry Fields

When entering a value in the entry fields for the Morse Speed (Words Per Minute) and the Tone Frequency (Hz) it is required to click the Set button for this to take effect. Pressing the RETURN key should also do the same.

Investigate how to implement this.
As a start there is a bind method available for Entry fields where the `<KeyPress>` attribute and a method to call can be specified. The method then needs to be defined. It will receive an event which has several properties. The state and keysym properties may be of interest to check which key or key combination that has been pressed. Investigate this and implement the described functionality so that the RETURN key works as intended.

In summary, it will look something like this:

**Binding a key press to an entry field**
`self.entry_field.bind("<KeyPress>", self.method)`

**Creating the method that will execute the key action**
<pre>def method(self, event):
    if event.state == ... and event.keysym == ... :
        .
        .
        .
</pre>

**Comment by callej** [comment]
**This task is completed**

It turns out that there already is a `"<Return>"` event that can be used. The solution becomes very simple. The Entry field only needs to bind to the `"<Return>"` event together with the function to be called. Here is the solution:

`self.mspeed_entry.bind("<Return>", self.set_mspeed)`
`self.mtone_entry.bind("<Return>", self.set_tone)`

<pre>
    def set_mspeed(self, e=None):
        try:
            wpm = max(min(int(self.ms_entry.get()), self.WPM_MAX), self.WPM_MIN)
            self.wpm.set(wpm)
            self.ms_entry.set(str(wpm))
        except ValueError:
            self.ms_entry.set(str(self.wpm.get()))
</pre>

<pre>
    def set_tone(self, e=None):
        try:
            freq = max(min(int(self.mt_entry.get()), self.TONE_MAX), self.TONE_MIN)
            self.tone.set(freq)
            self.mt_entry.set(str(freq))
        except ValueError:
            self.mt_entry.set(str(self.tone.get()))
</pre>

</br>

There are a ton of events and key symbols that can be used, making it extremely easy to achieve almost anything. Here are some useful references for the future:

- [keysyms ](https://tcl.tk/man/tcl8.6/TkCmd/keysyms.htm)
- [Events](https://www.tcl.tk/man/tcl/TkCmd/event.html)
- [TkDocs](https://tkdocs.com/)
- [Python interface to Tcl/Tk](https://docs.python.org/3/library/tkinter.html)
- [Graphical User Interfaces with Tk](https://docs.python.org/3/library/tk.html)

</br>

**Comment by callej** [comment]
**Regarding Events and what can be used.**

Try also:
`print(self.focus_get().event_info())`
